kfs-purap/src/main/resources/edu/arizona/kfs/module/purap/spring-purap.xml

```diff
@@ -182,6 +182,7 @@
 		<property name="businessObjectService" ref="businessObjectService" />
```

Comment:
This service definition bean should be in spring-tax.xml. None of the 1009 bean definitions should be in spring-purap.xml.